include/proj/coordinateoperation.hpp

```diff
@@ -1313,6 +1313,12 @@ class PROJ_GCC_DLL Conversion : public SingleOperation {
         const util::PropertyMap &properties, const common::Angle &centerLat,
         const common::Angle &centerLong, const common::Length &falseEasting,
         const common::Length &falseNorthing);
+    
+    PROJ_DLL static ConversionNNPtr createS2(
```

Comment:
unless you really need in the C++ API, this is not strictly needed and could be removed